### PR TITLE
Updated Projects Page

### PR DESCRIPTION
**Pull request by arfazca**
### GitHub Pull Repository Documentation Task

- [x] Update project statuses in the repository:
    - Add status descriptions: Active, Maintained, Complete, Inactive, etc
    - Add missing statuses like “Project maintained by other groups” or “Needs update”
    - Include a disclaimer: *“Meeting times are for Fall 2024”* to avoid confusion
- [x] Collect project information from admins (e.g., meeting time, description, tech stack) during the next admin meeting

### CODE REVIEW

File: pages/projects.tsx [hunk 1]
```diff
@@ -96,6 +96,32 @@ const projects = [
       lastUpdate: "Summer 2024",
       term: "Summer 2024"
     }
+  },
+  {
+    id: 7,
+    fields: { slug: "https://github.com/VikeLabs/study-sprints" },
+    frontmatter: {
+      title: "Study Sprints",
+      description: "An app for better time management for study sprints using 25-65 minute intervals and tracking past times.",
```
Comment (AmandaGAnderson) [inline pages/projects.tsx]:
```suggestion
      description: "A browser extension using the Pomodoro technique to encourage good study habits, studying in 26-65 minute intervals and tracking past habits.",
```

File: pages/projects.tsx [hunk 2]
```diff
@@ -96,6 +96,32 @@ const projects = [
...
+      lastUpdate: "Spring 2024",
+      term: "Spring 2024"
+    }
+  },
+  {
+    id: 8,
+    fields: { slug: "https://github.com/VikeLabs/StudySpaceFinder" },
+    frontmatter: {
+      title: "Study Space Finder",
+      description: "An application to find study spaces at the University of Victoria (UVic).",
```
Comment (AmandaGAnderson) [inline pages/projects.tsx]:
```suggestion
      description: "An application to find study spaces at the University of Victoria (UVic). Information is based on the class schedule gathered by UVic Course Scraper, and does not include room bookings by clubs or external groups.",
```

File: pages/projects.tsx [hunk 3]
```diff
@@ -64,7 +64,7 @@ const projects = [
     frontmatter: {
       title: "ESS Locker Registration",
       description: "A website for registration and management of lockers, designed with mobile-first functionality.",
```
Comment (AmandaGAnderson) [inline pages/projects.tsx]:
```suggestion
      description: "A website for registration and management of lockers, designed with mobile-first functionality. Now maintained by the UVic ECSS.",
```

**Review by AmandaGAnderson — CHANGES_REQUESTED** [review]
Thanks for updating this! I have a couple suggestions for the project descriptions, but otherwise it looks good to me. It even works on mobile. We'll share it with everyone during the admin meaning and create another PR if further changes are needed.

**Comment by arfazca** [comment]
> Thanks for updating this! I have a couple suggestions for the project descriptions, but otherwise it looks good to me. It even works on mobile. We'll share it with everyone during the admin meaning and create another PR if further changes are needed.

@AmandaGAnderson I've updated the project descriptions with your suggestions! I can create another PR if further changes are needed (I think we only need the updated meeting times). Otherwise, I will merge it soon!

**Review by AmandaGAnderson — APPROVED** [review]
Looks great!

**Pull request merged**